<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="orderDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>461</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ORDERS...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>30</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAguaDeBotella">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AGUA
DE
BOTELLA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonJugo">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JUGO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTe">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TÃ‰</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTortillas">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TORTILLAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum7">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum8">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum9">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum4">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum5">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum6">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum1">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum2">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum3">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNum0">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConfiguration">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totalAmountDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TOTAL $ 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPreviousTicket">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>930</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNextTicket">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>930</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMilanesa">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MILANESA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAlbondigas">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ALBÃ“NDIGAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCarneDeshebrada">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CARNE
DESHEBRADA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMole">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MOLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPechuga">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PECHUGA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPollo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>POLLO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChicharron">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CHICHARRÃ“N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBistec">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BISTEC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRefresco">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REFRESCO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuesadillaChica">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUESADILLA
CHICA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuesadillaCombinada">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUESADILLA
COMBINADA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCafe">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAFÃ‰</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuesadilla">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QUESADILLA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLicuado">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LICUADO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAguaFresca">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AGUA
FRESCA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTacoChico">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TACO
CHICO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHuevos">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HUEVOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChilaquiles">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CHILAQUILES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSincronizada">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>310</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SINCRONIZADA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBurritos">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>460</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BURRITOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateNewTicket">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>930</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteCurrentTicket">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>930</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ticketHeader">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ticketFooter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>872</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="DigitalClock" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1729</x>
      <y>930</y>
      <width>151</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QWidget</extends>
   <header>DigitalClock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
